cmd/server/main.go

```diff
@@ -81,16 +82,20 @@ func main() {
 	}
 
 	// Setup database connection
-	db, err := database.New(logger, os.Getenv("DATABASE_TYPE"))
+	conf := config.New()
+	err := conf.Load()
+	if err != nil {
```

Comment:
one-line if-else statements are a bit divided in the Go community:
Check out: https://www.calhoun.io/one-liner-if-statements-with-errors/

I guess it's more of a stylistic thing.

Reply:
I wouldn't suggest that one should _always_ use this convention, however it would make plenty of sense here, where the var `err` isn't actually needed to exist outside the scope of this `if` block.